src/labels/CustomLabels.labels

```diff
@@ -1871,7 +1871,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Sign on Payment and Allocation amount must match(Both positive or both negative)</shortDescription>
-        <value>Negative Allocations are only allowed in negative Payments and positive Allocations are not allowed in negative Payments.</value>
+        <value>Negative Allocations are only allowed in negative Gifts and positive Allocations are not allowed in negative Gifts.</value>
```

Comment:
@melissabarber I was under the impression that the main issue was that this error message shows up for Opportunities, but we're calling out "Payments" in the text and thus the switch to "Gift". Seems like maybe "Donations" might be the more standard word to use in these scenarios as per Laura Meerkatz in the "🐝 QA" channel.

Reply:
@psadie Just had a quick call with Ali to confirm the different conversations about this. As discussed hither and thither, this error message will be displayed in relation to Payments and Opportunities; therefore, it should remain ambiguous. Per Laura's suggestion, the typical ambiguous language for both Payments and Opportunities is "donations". Please update "Gifts" to be "donations" with a lower case d. I think this will help indicate we're not referring to a specific object named Donations. :)

Reply:
Awesome, thank you @melissabarber (and everyone who chimed in) for the update and for taking the time. I'll do my best to have more context around my WIs in the future. I'll have that update up shortly

Reply:
Great work @psadie!